### PR TITLE
Added "socketplane agent restart" (issue - enhancement #82)

### PR DESCRIPTION
**Pull request by botchagalupe**
Related to #82 

Adds a restart to the socketplane agent command...

e.g., 

socketplane agent restart

Signed-off-by: John Willis <botchagalupe@gmail.com>

### CODE REVIEW

File: scripts/socketplane.sh [hunk 1]
```diff
@@ -432,7 +432,9 @@ container_start() {
     cName=$(docker inspect --format='{{ .Name }}' $cid)
 
     json=$(curl -s -X GET http://localhost:6675/v0.1/connections/$cid)
-    result=$(echo $json | sed 's/[,{}]/\n/g' | sed 's/^".*":"\(.*\)"/\1/g' | awk -v RS="" '{ print $6, $7, $8, $9, $10 }')
+    result=$(echo $json | sed 's/[,{}]/\n/g' | sed 's/^".*":"\(.*\)"/\1/g' | awk -v RS="" '{ print $7, $8, $9, $10, $11 }')
```
Comment (mavenugo) [inline scripts/socketplane.sh]: This will result in a conflict due to #84 having merged already

File: scripts/socketplane.sh [hunk 2]
```diff
@@ -432,7 +432,9 @@ container_start() {
...
+    
+echo $json | sed 's/[,{}]/\n/g' | sed 's/^".*":"\(.*\)"/\1/g' | awk -v RS="" '{ print $6, $7, $8, $9, $10, $11 }'
```
Comment (mavenugo) [inline scripts/socketplane.sh]: Should this echo also needs an update as in #84 ?

**Comment by botchagalupe** [comment]
Dang... will fix

**Comment by botchagalupe** [comment]
Third time's a charm.. #90 should have it right...

**Pull request closed without merging**